.storybook/foundations/corner-rounding/corner-rounding.stories.js

```diff
@@ -0,0 +1,75 @@
+// Import the component markup template
+import { html } from "lit";
+import { Template } from './template';
+
+export default {
+	title: "Foundations/Corner Rounding",
+	description:
+		"Corner rounding is a foundation that shows the different border-radius tokens that can be applied to a component.",
+	component: "Corner Rounding",
+	args: {
+		rootClass: "spectrum-Foundations-Example-CornerRounding",
+	},
+  tags: ['foundation'],
```

Comment:
The `foundation` tag to hide the story from the sidebar so the primary function of the story is just to exist within the Corner Rounding Foundations doc.

I think it might be possible to make an .mdx file with the table similar to the component size tokens, but incorporating the CSS appropriately was a bit of a challenge when I tried this approach.